Solutions/NXLogDnsLogs/Parsers/ASimDnsMicrosoftNXLog.txt

```diff
@@ -7,7 +7,14 @@
 // Function usually takes 10-15 minutes to activate.
 // You can then use this function from any other queries (e.g. ASimDnsMicrosoftNXLog | take 10).
 // Reference: Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
-let ASimDnsMicrosoftNXLog = view () {
```

Comment:
@jszigetvari-nxlog  thankyou for your suggestion regarding modifying this solution parser.  We can use this parser to query ASIM function. Please replace this parser with the following query:

_Im_Dns
| where EventVendor == "Microsoft" and EventProduct == "DNS Server"

Reply:
Looks good